packages/tracing-internal/src/browser/metrics/index.ts

```diff
@@ -547,3 +530,41 @@ function setResourceEntrySizeData(
     data[dataKey] = entryVal;
   }
 }
+
+/**
+ * Add ttfb information to measurements
+ *
+ * Exported for tests
+ */
+export function _addTtfbToMeasurements(
+  _measurements: Measurements,
+  responseStartTimestamp: number | undefined,
+  requestStartTimestamp: number | undefined,
+  transactionStartTime: number | undefined,
+): void {
+  // Generate TTFB (Time to First Byte), which measured as the time between the beginning of the transaction and the
+  // start of the response in milliseconds
+  if (typeof responseStartTimestamp === 'number' && transactionStartTime) {
```

Comment:
l: We can maybe save a few bytes by checking `typeof responseStartTime !== 'number'` above and returning early, as we now repeat this below as well. but maybe also doesn't make any difference 😅